### ISSUE TITLE
MultiMap stats are missing memory usage

### ISSUE THREAD

**Issue by tolitius** [hazelcast/hazelcast#4653]
https://github.com/hazelcast/hazelcast/blob/master/hazelcast-documentation/src/ClusteredREST.md#retrieve-multimap-information

```json
  "ownedEntryMemoryCost":0
  "backupEntryMemoryCost":0
  "heapCost":0
```

Memory usage should be populated to be consistent with IMap.

**Comment by pveentjer** [comment]
Can you create a unit tests that shows it is actually broken. This saves us a lot of time.

And which HZ version are you using?

**Comment by metanet** [comment]
This issue is related to #4497 and fixed by #4691
